addon/components/ui-button/component.ts

```diff
@@ -15,9 +17,10 @@ export default class UiButton extends Component {
 
   onClick?: Function | null = null;
   label?: string = "";
-  appearance?: string = "default"; //Enum: default, strong, minimal
-  intent?: string = "none"; //Enum: none, success, warning, danger
-  size?: string = "default"; //Enum: small, large
+  appearance?: "default" | "strong" | "minimal" = "default";
+  width?: string; //Accepts any standard CSS width measurement
```

Comment:
Super pedantic but we can change to
`//Accepts any standard CSS width value`
:)